lalrpop/src/normalize/lower/mod.rs

```diff
@@ -176,7 +176,7 @@ impl<'s> LowerState<'s> {
 
         read_algorithm(&grammar.annotations, &mut algorithm);
 
-        let mut all_terminals: Vec<_> = self
+        let all_terminals: Vec<_> = self
```

Comment:
Why this change?

Reply:
mut was not necessary

Reply:
@Destroyerrrocket As I understand, he is asking about the removed sort below.

Reply:
Yep, that sort caused problems while trying to match str&. Tbf, it's been a while and I don't remember all the details, but it gave me problems...